pkg/render/intrusion_detection.go

```diff
@@ -201,9 +210,22 @@ func (c *intrusionDetectionComponent) Objects() ([]client.Object, []client.Objec
 			c.adAPIAccessClusterRole(),
 			c.adAPIAccessRoleBinding(),
 		)
+
+		configureADStorage := c.cfg.IntrusionDetection.Spec.AnomalyDetection.StorageType == operatorv1.PersistentStorageType
+
+		if configureADStorage {
+			// ignore all fields if it's set to using default ephermeal storage
```

Comment:
This comment seems out of place (and contains a typo)

Reply:
https://github.com/tigera/operator/pull/2094/commits/66c82b2c34b018e5472ab22a1ba6ad27b96bcc62